BetaCameras/Sick.VisionaryT.Pro/CameraModel.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Newtonsoft.Json;
+
+namespace MetriCam2.Cameras
```

Comment:
Please don't put internal classes in such a generic namespace.
Suggestion: `MetriCam2.Cameras.Internal.Sick`, as in the Visionary-T implementation

Reply:
Done.